deploy/push_to_rdss_willow.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+cd /home/willow/willow
+git push -u rdss-willow master
```

Comment:
should this set the `rdss-willow-master` remote on the previous line?

If the repo's just been (re) cloned for example, this will fail.

It's probably OK to publish the URLs to those repos? Some are in rspec comments already.